addon/components/polaris-visually-hidden.js

```diff
@@ -18,9 +14,8 @@ export default class PolarisVisuallyHidden extends Component {
    * in which case the block content will be used
    * instead of `text`
    *
-   * @type {string}
-   * @default null
+   * @type {String}
    * @public
    */
-  text = null;
+  text;
```

Comment:
What are your thoughts about needing these prop declarations, especially when we don't need default values (like here)?

Reply:
I'm flexible, left it for now
Wouldn't mind cleaning them up, but I thought that since it's already there, no harm